frontend/src/main.ts

```diff
@@ -11,10 +11,10 @@
 
 import { createApp } from 'vue';
 import Root from '@/App.vue';
-/* eslint-disable no-restricted-imports */
+// eslint-disable-next-line no-restricted-imports
 import { createRemote, i18n, router, vuetify } from '@/plugins';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
I agree for the other changes, but this one being multiline, i think it's better to use the previous method to disable and reenable later

Reply:
Here's my case for preferring single line vs a block:
1. Currently it's the same amount of comment lines, so there's no code savings really
2. The blocks shouldn't get much larger, since that would indicate we're making a lot of exceptions to our linting rules, which in general is bad
3. We can always switch these to a disable/enable scheme if they do get much larger

Let me know if I've swayed you or if you'd still like these reverted :)

Reply:
Thanks for that explanation @ferferga. I'd also like to clarify my position a bit:
Linting rules exist to enforce code standards. There will always be exceptions to those rules, which is why these exist. However, we should try to limit those exceptions, and adding individual comments is a clear way to show that a specific line should be ignored. Disable/enable should be saved for large sections where linting needs to be diabled, and these just don't exist in our code base.

That all being said, my opinion here that single is better than block, especially in this case, isn't strongly enough held that I'd want to prevent the rest of the PR from getting merged, so I'll revert this for now :)

I'm not sure if there's a linting rule to enforce certain line counts, but having a lint rule for lint exceptions is an interesting thought :D.

Reply:
@aweebs There are (https://mysticatea.github.io/eslint-plugin-eslint-comments/) and we use some of them. In fact, the use of enable-disable with the rule specified is enforced by that plugin.